scripts/installer/windows/scripts/pfs.bat

```diff
@@ -2,4 +2,12 @@
 setlocal
 
 set MAIN_EXE=%~dp0.\pfcli.exe
-"%MAIN_EXE%" pfs %*
\ No newline at end of file
+REM Check if the first argument is 'start'
```

Comment:
![image](https://github.com/microsoft/promptflow/assets/47586720/73339965-bc4c-4b9d-b001-dcfd09e8b201)
Can we not print the first two lines log?

Reply:
These two lines is printed because vbs. Need investigate whether we can remove it.

Reply:
removed